src/components/ResetPassword/ResetPasswordModal/ResetPasswordModal.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import CommonModal from '../../../common/CommonModal/CommonModal';
+import ResetPasswordModalContent from './ResetPasswordModalContent';
+import styles from './ResetPasswordModal.module.css';
+import CheckMark from '../../../assets/BlueCheckMark.png';
```

Comment:
Can we use the CheckCircleIcon in Chakra Icons here instead of a png?
https://chakra-ui.com/docs/components/media-and-icons/icon